api/v1alpha1/databasecluster_types.go

```diff
@@ -75,6 +75,9 @@ type DatabaseClusterStatus struct {
 //+kubebuilder:subresource:status
 
 // DatabaseCluster is the Schema for the databaseclusters API
+// +kubebuilder:printcolumn:name="Engine",type=string,JSONPath=`.spec.engine`
+// +kubebuilder:printcolumn:name="DO name",type=string,JSONPath=`.spec.name`
```

Comment:
I think `DO name` might be confusing for users. I'd suggest maybe `Cluster name` instead.

Reply:
Agree